index.html

```diff
@@ -169,6 +165,30 @@ <h2>Producing</h2>
         </p>
       </li>
 
+
+      
+
+      <li>
+        <p>
+          <a data-cite="CWT_HEADER#section-3">iss</a> MAY be present in <a data-cite="RFC9052#name-common-cose-header-paramete">COSE Headers</a>, 
```

Comment:
Reusing the `iss` claim from CWTs means adopting its definition:
> The "iss" (issuer) claim has the same meaning and processing rules as the "iss" claim defined in [Section 4.1.1 of [RFC7519]](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1)

And RFC 7519 says:
> "The "iss" (issuer) claim identifies the principal that issued the JWT."

To me this means that using that claim implies that there must be a CWT somewhere. Isn't that confusing?

Reply:
yes, but I think the same issue applies to any presence of `iss` in a header, when `iss` does not occur in a claims set (AKA all SCITT use cases where the payload is not a claim set, aks all SCITT use cases where the signed statement is not a CWT).

Reply:
Right, and I think that's why so far we were using a separate/new header parameter for the issuer, not reusing any from JWT/CWT.

Reply:
Where do I cite your `iss` header parameter... if we are not using the same tag from the CWT Header draft.

Reply:
Filed for continued discussion: https://github.com/transmute-industries/vc-cose/issues/8